jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -167,7 +186,7 @@ teardown() {
     export RHPAM_JTA="true"
     configure_EJB_Timer_datasource >&2
     echo "EJBTimer url is ${EJB_TIMER_XA_CONNECTION_PROPERTY_Url} " >&2
-    echo "RHPAM url is ${RHPAM_URL} " >&2
+    echo "RHPAM_XA_CONNECTION_PROPERTY_Url is ${RHPAM_XA_CONNECTION_PROPERTY_Url} " >&2
```

Comment:
Any reason to update it?
as it is a nonXA datasource RHPAM_XA_CONNECTION_PROPERTY_Url should not be set.